tweets/migrations/0004_tweet_updated_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.3 on 2024-07-04 12:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tweets', '0003_rename_retweet_tweet_original_tweet'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tweet',
+            name='updated_at',
+            field=models.DateTimeField(auto_now=True, null=True),
+        ),
+    ]
```

Comment:
Combine all the migration to one.

I will reset my local DB

Reply:
done